decidim-core/spec/commands/decidim/invite_user_to_private_participatory_space_spec.rb

```diff
@@ -83,7 +83,7 @@ module Decidim
     context "when the user is a member of the private space already" do
       let!(:organization) { create :organization }
       let!(:private_participatory_space) { create(:participatory_process, organization: organization, private_space: true) }
-      let!(:user) { create :user, organization: organization, invitation_accepted_at: Time.now }
+      let!(:user) { create :user, organization: organization, invitation_accepted_at: Time.zone.now }
```

Comment:
Hi! in Rails you can use `Time.current`, which is the same as `Time.zone.now`, and mostly used in Decidim. Cheers 😁

https://thoughtbot.com/blog/its-about-time-zones

Reply:
Alright! Thanks for the info. Done!